omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ProgramController1_9Test.java

```diff
@@ -153,6 +154,23 @@ public void shouldRetireAProgram() throws Exception {
 		Assert.assertEquals("some good reason", retiredProgram.getRetireReason());
 	}
 	
+	@Test
+	public void shouldUnRetireAProgram() throws Exception {
+		Program program = service.getProgramByUuid(getUuid());
+		program.setRetired(true);
+		program.setRetireReason("random reason");
+		program = service.saveProgram(program);
+		Assert.assertTrue(program.isRetired());
+		
```

Comment:
Can you first make a REST call and confirm that it is indeed retired?

Reply:
I have used `Service.get.......ByUuid()` which is implicitly called on any GET request for a given resource as suggested somewhere on this PR.